build.boot

```diff
@@ -101,14 +101,19 @@
 ;; these edn files can hold additional information about the page such
 ;; as it's permalink identifier (`:page` key) or the page's title etc.
 
-(deftask test! []
+(deftask test!
+  "Run tests."
+  []
   (set-env! :source-paths #(conj % "test")
-            :dependencies #(into % '[[cljs-react-test "0.1.4-SNAPSHOT" :scope "test" :exclusions [cljsjs/react-with-addons]]
-                                     [cljsjs/react-with-addons "0.14.7-0"]]))
+            :dependencies #(into % '[[cljs-react-test "0.1.4-SNAPSHOT" :scope "test" :exclusions [cljsjs/react-with-addons]]]))
   (test-cljs :js-env :phantom
              :exit? true
              :update-fs? true
-             :namespaces #{"test.open-company-web.*"}))
+             :namespaces #{"test.open-company-web.*"}
+             :cljs-opts {:optimizations :whitespace
+                         :foreign-libs [{:provides ["cljsjs.react"]
+                                         :file "https://cdnjs.cloudflare.com/ajax/libs/react/15.4.2/react-with-addons.js"
+                                         :file-min "https://cdnjs.cloudflare.com/ajax/libs/react/15.4.2/react-with-addons.min.js"}]}))
```

Comment:
That's really the point @martinklepsch.. i found that using our usual configuration for tests was broken when upgrading to react 15.4.2. I was getting the same error of the issue i linked you.
So i tried the solution. The point is that it never uses the :advanced build config so the minified file is never used (if i'm right the minified version of the foreign libs are used only with advanced compilation).
In the end their solution worked but only because i switched from cljsjs to foreign lib for react-with-addons when testing (and it works only w/ simple and white-space configuration, i didn't test it w/ advanced).
It's a little messy i know and i really didn't want to spend to much time on this since we don't have too much test in place for our WEB project so when i found something that worked i stopped looking into it.
If you can find something better that works please let me know.